codecs/oxipng/Cargo.toml

```diff
@@ -12,7 +12,19 @@ crate-type = ["cdylib"]
 oxipng = { version = "3.0.0", default-features = false }
 wasm-bindgen = "0.2.64"
 log = { version = "0.4", features = ["release_max_level_off"] }
+rayon = { version = "1.3.0", optional = true }
+once_cell = { version = "1.3.1", optional = true }
+crossbeam-channel = { version = "0.4.2", optional = true }
+
+[patch.crates-io]
+oxipng = { git = "https://github.com/RReverser/oxipng", branch = "crossbeam" }
```

Comment:
Oh dear 😅  Why is this necessary? Are we going to maintain a fork long-term now?

Reply:
No, I'll upstream changes later, but for now wanted to have flexibility. It's not the first time we're pointing to my fork, did the same in the initial version of OxiPNG while waiting for upstream :)

Reply:
Can you add a comment with the upstream issue/PR so we know when we can replace the fork with the original again?

Reply:
I don't have one ready yet, didn't get to the "upstreaming" part of this feature.

Reply:
Done - created PR [here](https://github.com/shssoichiro/oxipng/pull/327) and added as a comment.